doc/source/serve/doc_code/deployment_graph_control_flow_based_on_user_inputs.py

```diff
@@ -0,0 +1,36 @@
+import ray
+from ray import serve
+from ray.experimental.dag.input_node import InputNode
+
+
+ray.init()
+serve.start()
+
+
+@serve.deployment
+class Model:
+    def __init__(self, weight):
+        self.weight = weight
+
+    def forward(self, input):
+        return input + self.weight
+
+
+@serve.deployment
+def combine(value_refs, combine_type):
+    values = [ray.get(value_ref) for value_ref in value_refs]
```

Comment:
```suggestion
def combine(value_refs, combine_type):
    values = await asyncio.gather(**value_refs)
```

Reply:
content looks good to me and i second this, using async function with asyncio.gather() is basically always more favorable than ray.get() especially in serve, since ray.get() is blocking.